From: Maciek Borzecki <maciek.borzecki@gmail.com>
Date: Tue, 20 Sep 2016 21:24:45 +0200
Subject: nss: install nss modules to ${rootlibdir}

NSS modules (libnss_*.so.*) need to be installed into
${rootlibdir} (typically /lib) in order to be used. Previously, the
modules were installed into ${libdir}, thus usually ending up in
/usr/lib, even on systems where split usr is enabled, or ${libdir} is
passed explicitly.

Signed-off-by: Maciek Borzecki <maciek.borzecki@gmail.com>
---
 Makefile.am | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 6aebca4..c668187 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -157,6 +157,7 @@ nodist_zshcompletion_DATA = $(nodist_zshcompletion_data)
 endif
 udevlibexec_PROGRAMS =
 gperf_gperf_sources =
+rootlib_LTLIBRARIES =
 
 in_files = $(filter %.in,$(EXTRA_DIST))
 in_in_files = $(filter %.in.in, $(in_files))
@@ -5021,7 +5022,7 @@ libnss_myhostname_la_LIBADD = \
 	libsystemd-internal.la \
 	libbasic.la
 
-lib_LTLIBRARIES += \
+rootlib_LTLIBRARIES += \
 	libnss_myhostname.la
 endif
 
@@ -5120,7 +5121,7 @@ libnss_mymachines_la_LIBADD = \
 	libsystemd-internal.la \
 	libbasic.la
 
-lib_LTLIBRARIES += \
+rootlib_LTLIBRARIES += \
 	libnss_mymachines.la
 
 endif
@@ -5419,7 +5420,7 @@ libnss_resolve_la_LIBADD = \
 	libbasic.la \
         -ldl
 
-lib_LTLIBRARIES += \
+rootlib_LTLIBRARIES += \
 	libnss_resolve.la
 
 systemd_resolve_SOURCES = \
